cityofaustin/atd-moped
# Refactor audit tables on `moped_proj_funding`

## Associated issues

This PR hopes to close https://github.com/cityofaustin/atd-data-tech/issues/15086.

## Testing

This has some DB changes, so this should be tested locally. 

**Steps to test:**

* Spin up the database, optionally replicating from production.
* Observe that migrations and metadata work.
* Spin up your editor.
* Pick a project, and add a funding source.
* Query that record in the DB and see that `created_at` and `created_by_user_id` are populated.
* Edit that record through the editor UI and observe that `updated_at` and `updated_by_user_id` are populated correctly in the `moped_proj_funding` record and that the parent project's record has its `updated_at` and `updated_by_user_id` fields populated. 
* Check out the permissions for `moped_proj_funding` in the hasura console.
* Check out the migration, and check out the down migration. Do they look correct?
* Check the changes to the event trigger for the funding table. Does it correctly no longer manipulate the audit fields in the project table?

---
#### Ship list
